stable/prometheus-mongodb-exporter/templates/deployment.yaml

```diff
@@ -58,6 +58,15 @@ spec:
           {{- toYaml .Values.resources | nindent 10 }}
         securityContext:
           {{- toYaml .Values.securityContext | nindent 10 }}
+      {{- if .Values.cacert.enabled }}
+        volumeMounts:
+          - name: {{ include "prometheus-mongodb-exporter.fullname" . }}-cacert
+            mountPath: /var/mongodb_exporter/config
```

Comment:
How does the mongo exporter process know to find certs at this path? Please add a section to README on how to use it if it requires some extra configuration like env variables or command line arguments.